src/code-templates/services/order-service/jest.config.js

```diff
@@ -99,7 +99,20 @@ module.exports = {
   // projects: undefined,
 
   // Use this configuration option to add custom reporters to Jest
-  // reporters: undefined,
+  reporters: [
+    "default",
+    [
+      "@jest-performance-reporter/core",
+      {
+        "errorAfterMs": 1000,
+        "warnAfterMs": 500,
+        "logLevel": "warn",
+        "maxItems": 5,
```

Comment:
What is `maxItems`?

Reply:
@rluvaton I would guess that it means how many slow tests it should show

Reply:
correct